ray-operator/controllers/ray/raycluster_controller.go

```diff
@@ -220,10 +220,33 @@ func (r *RayClusterReconciler) validateRayClusterStatus(instance *rayv1.RayClust
 	return nil
 }
 
+// Validation for invalid Ray Cluster configurations.
+func (r *RayClusterReconciler) validateRayClusterSpec(instance *rayv1.RayCluster) error {
```

Comment:
This function doesn't need to be the member function of `RayClusterReconciler`. You can open another PR to update `validateRayClusterStatus`.
```suggestion
func validateRayClusterSpec(instance *rayv1.RayCluster) error {
```